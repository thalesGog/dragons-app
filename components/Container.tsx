import styled from 'styled-components'

const Container = styled.main`
  max-width: 1024px;
  margin: 0 auto;
  text-align: center;
  height: 100vh;
  padding-left: 8px;
  padding-right: 8px;
  display: flex;
`

export default Container
