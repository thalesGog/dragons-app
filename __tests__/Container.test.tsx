import React from 'react'
import {render} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Container from '../components/Container'

test('Render Container component', async () => {
  render(<Container />)
})
