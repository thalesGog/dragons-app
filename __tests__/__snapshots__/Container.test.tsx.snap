// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Container component 1`] = `
.c0 {
  max-width: 1024px;
  margin: 0 auto;
  text-align: center;
  padding-left: 8px;
  padding-right: 8px;
  padding-bottom: 80px;
}

<div>
  <main
    class="c0"
  />
</div>
`;
